plans/provision_agent.pp

```diff
@@ -42,7 +42,7 @@
         # The pe_repo class for Ubuntu does not have the decimals in the version
         $platform = regsubst(deploy_pe::platform_tag($target_facts, true), '\.', '')
         run_command(
-          "/opt/puppetlabs/puppet/bin/puppet apply -e \"include pe_repo::platform::${platform}\"",
+          "/opt/puppetlabs/puppet/bin/puppet apply -e \"include pe_repo::platform::${platform}\"; package { 'pe-java11': }",
```

Comment:
Thanks for the PR! SUP-2149 is tracking the fix for this one. I am concerned about adding the `pe-java11` package since it will probably fail on PE versions with java8. How about making this conditional on the master's `pe_build` fact? We can use this command on anything greater than 2019.8.0

Also, what are your thoughts on disabling bulk plugin sync rather than the specific package name? I was thinking that the package version could change in future versions, but disabling bulk plugin sync may may be more stable. 

```
 puppet apply -e 'class{"pe_repo": enable_bulk_pluginsync => false, enable_windows_bulk_pluginsync => false }; include pe_repo::platform::el_7_x86_64'
```

Reply:
Yeah actually disabling bulk pluginsync would be a better solution